simulator/src/main/java/com/hazelcast/simulator/worker/WorkerCommandRequestProcessor.java

```diff
@@ -329,21 +327,17 @@ private Long process(GetOperationCountCommand command) throws Exception {
         }
 
         private List<String> process(GetStackTraceCommand command) throws Exception {
-            try {
-                String testId = command.testId;
-                final String testName = "".equals(testId) ? "test" : testId;
-                TestContainer<TestContext> test = tests.get(command.testId);
-                if (test == null) {
-                    LOGGER.warn("Can't get stacktraces, found no test with id " + command.testId);
-                    return null;
-                }
+            String testId = command.testId;
```

Comment:
There is no exception thrown within the called methods. And if so the `WorkerCommandRequestProcessorThread.run()` will catch it for us.